azure-pipelines/secure-development-tools.yml

```diff
@@ -9,3 +18,87 @@ steps:
   displayName: 'Run PoliCheck'
   inputs:
     targetType: F
+
+ ### Analyze managed and unmanaged binaries (exe, dll) for security vulnerabilities
+- task: BinSkim@3
+  displayName: Run BinSkim
+  inputs:
+    InputType: 'Basic'
+    Function: 'analyze'
+    AnalyzeTarget: '$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)/netstandard2.0/StreamJsonRpc.dll;$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)/netcoreapp2.1/StreamJsonRpc.dll'
+
+ ### Copy existing Roslyn Analyzer logs to be analyzed
+- task: RoslynAnalyzers@3
+  displayName: Run Roslyn Analyzers.
+  inputs:
+    copyLogsOnly: 'true'
+  continueOnError: True
+
+ ### Copy files for APIScan
+- task: CopyFiles@2
+  displayName: 'Copy Files for APIScan'
+  inputs:
+    SourceFolder: '$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)'
+    Contents: |
+      netcoreapp2.1\StreamJsonRpc.*
+    TargetFolder: $(Agent.TempDirectory)\APIScanFiles\netcoreapp2.1
+  condition: ne(variables['DisableAPIScan'], 'true')
+
+- task: CopyFiles@2
+  displayName: 'Copy Files for APIScan'
+  inputs:
+    SourceFolder: '$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)'
+    Contents: |
+      netstandard2.0\StreamJsonRpc.*
+    TargetFolder: $(Agent.TempDirectory)\APIScanFiles\netstandard2.0
+  condition: ne(variables['DisableAPIScan'], 'true')
+
+ ### Run APIScan
+- task: APIScan@1
+  displayName: Run APIScan
+  inputs:
+    softwareFolder: $(Agent.TempDirectory)\APIScanFiles
+    softwareName: 'Microsoft.VisualStudio.LanguageServer'
+    softwareVersionNum: '$(Build.BuildId)'
+    isLargeApp: false
+    toolVersion: LatestPreRelease
```

Comment:
This `toolVersion` seems most likely to be unstable. Can we use a latest stable version or even better just *pin* the version so that it doesn't suddenly start failing when they push an update?